beamer/agent/chain.py

```diff
@@ -26,6 +26,8 @@
 
 POLL_PERIOD: float = 5
 
+CONFIRMATION_BLOCKS: int = 1
```

Comment:
Could you please make this configurable, per-chain, similar to what Manuel has done for the poll period?
We should have one global default which is used in case the chain-specific value is not defined.